src/gui/mzroll/eicwidget.h

```diff
@@ -257,7 +260,6 @@ public Q_SLOTS:
 	void cleanup();		//deallocate eics, fragments, peaks, peakgroups
 	void clearPlot();	//removes non permenent graphics objects
 	void findPlotBounds(); //find _minX, _maxX...etc
-	mzSlice visibleSamplesBounds();
```

Comment:
@saiful-elucidata  _visibleSampleBounds_ is not used as a slot. I think we should move this function to public and not in slots.

Reply:
Good catch. Moving to `public` methods.